### PR TITLE
Doc updates for 1.1 release

### PR DESCRIPTION
**Pull request by markxnelson**
This is doc updates for the 1.1 release.

### CODE REVIEW

File: docs-source/spring/content/development/envvars.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+`MP_LRA_PARTICIPANT_URL`               | `http://customer32.application.svc.cluster.local:8080` | This is the participant URL which is required when using Eclipse Microprofile Long Running Actions in your application. 
+`eureka.client.register-with-eureka`   | `true`         | This tells the Eureka discovery client to register with the Eureka server.
+`eureka.client.fetch-registry`         | `true`         | This tells the Eureka discovery client to make a local copy of the registry by fetching it from the Eureka server.
+`eureka.client.service-url.defaultZone` | `http://eureka.eureka:8761/eureka` | This is the default zone for the Eureka discovery client.
+`zipkin.base-url`                      | `http://jaegertracing-collector.observability:9411` | This is the URL of the Zipkin-compatible trace collector which can be used by your application to send trace data to the platform.
+`otel.exporter.otlp.endpoint`          | `http://open-telemetry-opentelemetry-collector.open-telemetry:4317` | This is the URL of the OpenTelemetry (OTLP protocol) trace collector which can be used by your application to send trace data to the platform.
```
Comment (andytael) [inline docs-source/spring/content/development/envvars.md:22]: Wrong URL, should be http://open-telemetry-opentelemetry-collector.open-telemetry:4318/v1/traces

File: docs-source/spring/content/release-notes/_index.md [hunk 2]
```diff
@@ -2,6 +2,44 @@
...
+* A preview of a new "Spring Operations Center" feature is added.
+* User Management using the CLI and REST Endpoints.
+* Bug fixes.
+* xyz
+
+The following components were added:
+
+| Component                               | Version   | Description                                     |
+|-----------------------------------------|-----------|-------------------------------------------------|
+| Oracle Database Observability Exporter  | 1.1.2     | Provides observability for the Oracle Database. |
```
Comment (andytael) [inline docs-source/spring/content/release-notes/_index.md:19]: Isn't this version 1.2.0

File: docs-source/spring/content/development/envvars.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+`MP_LRA_COORDINATOR_URL`               | `http://otmm-tcs.otmm.svc.cluster.local:9000/api/v1/lra-coordinator` |  This is the URL for the transaction manager which is required when using Eclipse Microprofile Long Running Actions in your application. 
+`MP_LRA_PARTICIPANT_URL`               | `http://customer32.application.svc.cluster.local:8080` | This is the participant URL which is required when using Eclipse Microprofile Long Running Actions in your application. 
+`eureka.client.register-with-eureka`   | `true`         | This tells the Eureka discovery client to register with the Eureka server.
+`eureka.client.fetch-registry`         | `true`         | This tells the Eureka discovery client to make a local copy of the registry by fetching it from the Eureka server.
+`eureka.client.service-url.defaultZone` | `http://eureka.eureka:8761/eureka` | This is the default zone for the Eureka discovery client.
+`zipkin.base-url`                      | `http://jaegertracing-collector.observability:9411` | This is the URL of the Zipkin-compatible trace collector which can be used by your application to send trace data to the platform.
```
Comment (andytael) [inline docs-source/spring/content/development/envvars.md:21]: Wrong URL, should be http://jaegertracing-collector.observability.svc.cluster.local:9411/api/v2/spans

File: docs-source/spring/content/development/envvars.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+`spring.datasource.password`:          | set to the key `db.password` in secret `customer32-db-secrets` | This sets the (regular) user's password for your applciation to use to connect to the database (if you use JPA or JDBC in your application).
+`DB_SERVICE`                           | set to the key `db.service` in secret `customer32-db-secrets` | This sets the database service name (the TNS name) for your applciation to use to connect to the database (if you use JPA or JDBC in your application).
+`spring.datasource.url`                | `jdbc:oracle:thin:@$(DB_SERVICE)?TNS_ADMIN=/oracle/tnsadmin` | This sets the data source URL for your applciation to use to connect to the database (if you use JPA or JDBC in your application).
+`CONNECT_STRING`                       | `jdbc:oracle:thin:@$(DB_SERVICE)?TNS_ADMIN=/oracle/tnsadmin` | This sets the data source URL for your applciation to use to connect to the database (if you use JPA or JDBC in your application).
```
Comment (andytael) [inline docs-source/spring/content/development/envvars.md:30]: Typo: applciation

**Review by andytael — APPROVED** [review]

**Review by pasimoes — APPROVED** [review]

**Pull request merged**